app/src/main/res/layout/selection_mode_options_layout.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<GridLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/selection_mode_options"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:rowCount="@integer/selection_modes_rows"
+    android:columnCount="@integer/selection_modes_columns">
+
+    <com.willowtree.vocable.customviews.VocableConstraintLayout
+        android:id="@+id/head_tracking_container"
+        android:layout_width="0dp"
+        android:layout_height="@dimen/selection_mode_button_height"
+        android:layout_rowWeight="1"
+        android:layout_columnWeight="1"
+        android:layout_marginEnd="@dimen/selection_mode_button_end_margin"
+        android:layout_marginBottom="@dimen/selection_mode_buttom_bottom_margin"
+        android:background="@drawable/settings_group_background"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <TextView
+            android:layout_width="wrap_content"
```

Comment:
It'll probably be good to go ahead an make a style for this `TextView` and the `SwitchCompat` below since future settings options will share the same properties.